fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1434,6 +1434,192 @@ module.exports = function(config, DB) {
         )
 
         test(
+          'securityEvents',
+          function (t) {
+
+            function securityEvents(suite) {
+              return suite.setUp().then(function () {
+                delete suite.setUp
+
+                var names = Object.keys(suite)
+
+                function run () {
+                  var unit = names.shift()
+
+                  if (unit) {
+                    return suite[unit]().then(run)
+                  }
+                }
+
+                return run()
+              }).done(function () {
+                t.end()
+              })
+            }
+
+            function createSession (id, session) {
+              return db.createSessionToken(id, session).catch(function (e) {
+                t.fail('createSession ' + id.toString('hex') + ' failed: ' + e)
+              })
+            }
+
+            function verifySession (id, uid) {
+              return db.verifyTokens(id, { uid: uid }).catch(function (e) {
+                t.fail('verifySession ' + id.toString('hex') + ' failed: ' + e)
+              })
+            }
+
+            function deleteSession (id) {
+              return db.deleteSessionToken(id).catch(function (e) {
+                t.fail('deleteSession ' + id.toString('hex') + ' failed: ' + e)
+              })
+            }
+
+            function insert (uid, addr, name, session) {
+              return db.createSecurityEvent({
+                uid: uid,
+                ipAddr: addr,
+                name: name,
+                sessionTokenId: session
+              })
+            }
+
+            function query (uid, addr, name, cb) {
+              return function () {
+                return db.securityEvents({
+                  uid: uid,
+                  ipAddr: addr,
+                  name: name
```

Comment:
AFAICT `db.securityEvents` does not accept a `"name"` option.

Also, the `query` function is defined here as `query(uid, addr, name, cb)` but below is called as `query(uid, addr, cb)`, which I think may lead to the callback function not actually being executed.